modules/gitops-argo-cd-installation.adoc

```diff
@@ -19,8 +19,8 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 . Click *Create* to configure the following parameters:
 
-.. For *Name*, enter `/`.
+.. By default the *Name* is set as *argocd*. Enter the *Name* of the instance to change the name.
 
-.. Create an external OS Route to access Argo CD server. Click *Server* -> *Route* and, for *Enable Route*, select *true*.
+.. Click *Server* -> *Route* and, for *Enable Route*, select *true*. By default, the server route is enabled. 
```

Comment:
> Click *Server* -> *Route* and, for *Enable Route*, select *true*. By default, the server route is enabled. 

Create an external OS Route to access Argo CD server by selecting **Server -> Route** and checking **enabled** (It is enabled by default)